examples/gameserverallocation.yaml

```diff
@@ -85,3 +85,34 @@ spec:
       mode: deathmatch
     annotations:
       map: garden22
+  # [Stage: Alpha]
+  # [FeatureFlag:CountsAndLists]
+  # `Priorities` configuration alters the order in which `GameServers` are searched for matches to the configured `selectors`.
+  #
+  # Priority of sorting is in descending importance. I.e. The position 0 `priority` entry is checked first.
+  #
+  # For `Packed` strategy sorting, this priority list will be the tie-breaker within the least utilised infrastructure, to ensure optimal
+  # infrastructure usage while also allowing some custom prioritisation of `GameServers`.
+  #
+  # For `Distributed` strategy sorting, the entire selection of `GameServers` will be sorted by this priority list to provide the
+  # order that `GameServers` will be allocated by.
+  # Optional.
+  # priorities:
```

Comment:
I don't think we _have_ to, since we don't iterate over all options for any other lists of items -- but we could?

I don't feel strongly either way - what do you think?

Reply:
No strong feelings, either way works.